packages/graphql-tool-utilities/src/config.ts

```diff
@@ -17,7 +18,7 @@ export function getGraphQLProjects(config: GraphQLConfig) {
 
   const project = config.getProjectConfig();
 
-  if (project) {
+  if (project && project.schemaPath) {
```

Comment:
without validation, a totally invalid project config actually returns a project instance. this check on the `schemaPath` prevents that from happening so we drop into our expected error thrown scenario.